Calmspace/MeditationCell.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  MeditationCell.swift
+//  Calmspace
+//
+//  Created by Admin on 18/11/2023.
+//
+
+import UIKit
+
+class MeditationCell: UITableViewCell {
+
+    @IBOutlet weak var customLabel: UILabel!
```

Comment:
I think that all the custom cells subclasses that are custom just because they have a label can be replaced with a standard `UITableViewCell.contentConfiguration`. This is a relatively new feature in UIKit presented on WWDC https://developer.apple.com/videos/play/wwdc2020/10027/